qube/create/templates/common_files/{{ cookiecutter.commonName }}/.github/workflows/nexus-publish.yml

```diff
@@ -5,8 +5,6 @@
 name: Nexus Package
 
 on:
-  release:
-    types: [created]
   push:
     branches:
       - development
```

Comment:
Does that mean that we only allow releases from development?

Reply:
it removes the conditional execution when a tag was created (which happens on Github upon a release). I have two conditional workflows, one that is triggered for development, one for releases.